### DISCUSSION TITLE
Want to do Functor.map(record.Functor, array.Functor) but can't find HKT implementation for Record

### DISCUSSION THREAD

**Discussion by ingun37** [gcanti/fp-ts#1623] [Q&A]
I want to compose Record Functor and Array Functor so I can have Record<Array<T>> Functor.
It seems `getFunctorComposition` is deprecated by `Functor.map`.
But `Functor.map` takes only `Functor` which is `HKT1` rather than `Kind1`.
But I can't find any `Functor` implementation of Record. I can't use `record.Functor` is because it's `Kind1`.
What can I do in this situation?

**Comment by gcanti** [comment]
Solved in v2.11.7
